tox.ini

```diff
@@ -136,12 +136,18 @@ deps =
   ansicolors==1.1.8
   pip==19.3.1
   redbaron==0.9.2
+  setuptools==42.0.2
+  wheel==0.33.6
```

Comment:
To confirm, this is where we pin the version of `setuptools` and `wheel` that we're going to ship with the Pex binary?

Reply:
No, this is where we pin the versions used when tox runs `pex.vendor`. The versions we actually vendor are in `__init__.py`.

Reply:
Might be worth adding a comment that these need to be in sync.

Reply:
They don't need to be in sync, they just need to be pinned versions that work with the version of pip used above which itself does not need to match our vendored pip version as it currently does not. That said, see above - I think we can eliminate this change by pruning more metadata files post install.